package.json

```diff
@@ -74,17 +74,19 @@
     "mocha": "^3.0.2",
     "performance-now": "^2.1.0",
     "phantomjs": "^1.9.13",
-    "protractor": "^5.4.2",
+    "protractor": "^5.4.4",
     "react": "^0.14.0",
     "react-dom": "^0.14.0",
     "react-router": "^1.0.0-rc3",
     "sanitize-html": "^1.15.0",
     "strip-json-comments": "^1.0.2"
   },
   "scripts": {
-    "test": "grunt test",
```

Comment:
I think we could leave this for other cases where you want to run both unit and e2e